add throttle to register route (#1194)
plentymarkets/plugin-io: meta/documents/changelog_en.md

- Solved a bug that manifested by redirecting the user to the homepage when the language was changed on order confirmation page.
- The wish list now preserves the order of the items on the list.

### Changed

> - The route to customer registration now contains a core-side limitation.

Reviewer: ```suggestion
- The customer registration route now contains an IP-based limitation in the core.
```
